app/views/candidate_interface/application_form/submit_success.html.erb

```diff
@@ -13,18 +13,18 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h2 class="govuk-heading-m">What happens next</h2>
-    <p class="govuk-body">
-      The Apply for teacher training service will send you an email to confirm the successful submission of your application.
-    </p>
+    <p class="govuk-body">The <%= service_name %> service will send you an email to confirm the successful submission of your application.</p>
 
     <h2 class="govuk-heading-m">References</h2>
-    <p class="govuk-body">We will also contact your referees to ask for your references. When we receive them, we will send them to your training provider. Your provider then has 40 working days to respond to your application.</p>
+    <p class="govuk-body">We will also contact your referees to ask for your references. When we receive them, we’ll add them to your application and send it to your training provider(s).</p>
```

Comment:
We can use some logic here to write `training provider` or `training providers` to avoid `training provider(s)`